Sources/VariantsCore/Factory/iOS/XcodeProjFactory.swift

```diff
@@ -224,7 +243,7 @@ private extension XcodeProjFactory {
          * If .xcconfig, set baseConfigurationReference to it
          */
         if file.lastComponent.contains(".xcconfig"), let fileReference = fileRef {
```

Comment:
Not really a change you did, but it seems we can:
```suggestion
        if file.extension == "xcconfig", let fileReference = fileRef {
```
https://github.com/kylef/PathKit/blob/master/Sources/PathKit.swift#L256